src/features/modal/studyCreation/RightSection.tsx

```diff
@@ -1,23 +1,52 @@
-import { css } from '@emotion/react';
+import { css, useTheme } from '@emotion/react';
+import {
+  type StudyCreationSectionProps,
+} from '@features/modal/studyCreation/StudyCreationModal';
+import { FormErrorMessage } from '@components/text/variants';
 import Button from '@/components/button';
 import Container from '@/components/container';
 import Switch from '@/components/switch';
 import { Paragraph } from '@/components/text';
 import Textarea from '@/components/textarea';
 import colorTheme from '@/styles/colors';
-import useStudyCreationStyle from './StudyCreation.styles';
 
-export default function RightSection() {
-  const { creationButtonStyle } = useStudyCreationStyle();
+export default function RightSection({
+  register,
+  formState: { errors, isValid },
+}: StudyCreationSectionProps) {
+  const theme = useTheme();
   return (
-    <Container direction="column" gap="10px" align="flex-start">
-      <Textarea label="스터디 설명" rows={14} resize="none" />
+    <Container direction="column" align="flex-start">
+      <Textarea
+        label="스터디 설명"
+        rows={14}
+        resize="none"
+        {...register('description', { ...validations.description })}
```

Comment:
```suggestion
        {...register('description', validations.description)}
```

spread operator (`...`) 사용하지 않아도 괜찮아보여용